test/configs/deployment-k8s-regional-rbd.yaml

```diff
@@ -2,9 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
-repo: https://github.com/ramendr/ocm-ramen-samples.git
+repo: https://github.com/elenager/ocm-ramen-samples.git
 path: subscription/deployment-k8s-regional-rbd
-branch: main
+branch: test
```

Comment:
For merging in drenv we need to remove this change. It will help to split it to another commit.